### ISSUE TITLE
How to setup own config / settings?

### ISSUE THREAD

**Issue by PixelT** [SublimeLinter/SublimeLinter-html-tidy#36]
For example - I have in my code:

<DIV class="test"></DIV>

I try a few ways to display uppercase tag as an error:

```
            "htmltidy": {
                "@disable": false,
                "args": [
                    "uppercase-tags","true"
                ],
                "excludes": []
            },
```

```
            "htmltidy": {
                "@disable": false,
                "args": [
                    "--uppercase-tags","true"
                ],
                "excludes": []
            },
```

```
            "htmltidy": {
                "@disable": false,
                "args": [
                    "--uppercase-tags true"
                ],
                "excludes": []
            },
```

but none of them work... what I'm doing wrong?

**Comment by cspiker** [comment]
Try using a string instead of an array:
```javascript
"htmltidy": {
    "@disable": false,
    "args": "--uppercase-tags true",
    "excludes": []
},
```
Alternatively, you can use a tidy config file and specify the path to the file in the args:
```javascript
"htmltidy": {
    "@disable": false,
    "args": "-config /path/to/tidy/config/file",
    "excludes": []
},
```
```
// tidy config file
uppercase-tags: true
```

**Comment by oktalk** [comment]
Sorry to resurrect this old issue, but I cannot for the life of me figure out how to use my own configuration file for HTML-tidy in Sublime Text 3.

Most other linters I've used (ESLint, SCSS-lint, etc.) all have either a YAML file or a JSON file that exist in the root of your project. I was able to find some [documentation](http://www.html-tidy.org/documentation/#using-a-configuration-file) on using a 'config.txt' file for [customizing the HTML-tidy settings](http://api.html-tidy.org/tidy/quickref_5.2.0.html#show-body-only), but I have no idea on how to configure Sublime to use that file.

If the above examples from @cspiker worked, where would that code exist in a project?
